identity-apps-core/apps/authentication-portal/src/main/webapp/fido2-auth.jsp

```diff
@@ -32,6 +32,31 @@
 <%@include file="includes/localize.jsp" %>
 <%@include file="includes/init-url.jsp" %>
 
+<%!
+    private boolean isMultiAuthAvailable(String multiOptionURI) {
+        boolean isMultiAuthAvailable = true;
```

Comment:
Add a new line after the method declaration, and check the other places as well.

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/5833